extensions/smallrye-openapi/deployment/src/test/java/io/quarkus/smallrye/openapi/test/OpenApiHttpRootDefaultPathTestCase.java

```diff
@@ -0,0 +1,41 @@
+package io.quarkus.smallrye.openapi.test;
+
+import org.hamcrest.Matchers;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.asset.StringAsset;
+import org.jboss.shrinkwrap.api.spec.JavaArchive;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import io.quarkus.test.QuarkusUnitTest;
+import io.restassured.RestAssured;
+
+public class OpenApiHttpRootDefaultPathTestCase {
+    private static final String OPEN_API_PATH = "/openapi";
+
+    @RegisterExtension
+    static QuarkusUnitTest runner = new QuarkusUnitTest()
+            .setArchiveProducer(() -> ShrinkWrap.create(JavaArchive.class)
+                    .addClasses(OpenApiResource.class)
+                    .addAsResource(new StringAsset("quarkus.http.root-path=/foo"), "application.properties"));
+
+    @Test
+    public void testOpenApiPathAccessResource() {
+        RestAssured.given().header("Accept", "application/yaml")
+                .when().get(OPEN_API_PATH)
```

Comment:
But… it's at `/openapi`? Shouldn't it be at `/foo/openapi`?

Reply:
Oh, I see. I wonder if that's nice or error-prone. I guess it's nice. Is it documented?

Won't we face people wondering why their browser says `/foo/openapi` exists and they write their test with `/foo/openapi` and get a 404?